assets/src/edit-story/components/panels/test/sizePosition.js

```diff
@@ -190,28 +200,53 @@ describe('Panels/SizePosition', () => {
     });
 
     it('should update width without lock ratio and empty value', () => {
-      const { getByTestId, pushUpdate } = renderSizePosition([defaultImage]);
-      fireEvent.click(getByTestId('lockRatio'));
+      const { getByTestId, pushUpdate } = renderSizePosition([
+        unlockAspectRatioElement,
+      ]);
       const input = getByTestId('width');
       fireEvent.change(input, { target: { value: '' } });
       expect(pushUpdate).toHaveBeenCalledWith({ width: '', height: 80 });
     });
 
     it('should update height without lock ratio and empty value', () => {
-      const { getByTestId, pushUpdate } = renderSizePosition([defaultImage]);
-      fireEvent.click(getByTestId('lockRatio'));
+      const { getByTestId, pushUpdate } = renderSizePosition([
+        unlockAspectRatioElement,
+      ]);
       const input = getByTestId('height');
       fireEvent.change(input, { target: { value: '' } });
       expect(pushUpdate).toHaveBeenCalledWith({ height: '', width: 100 });
     });
+
+    it('should update lock ratio to false for element', () => {
+      const { getByTestId, pushUpdate } = renderSizePosition([defaultImage]);
+      fireEvent.click(getByTestId('lockRatio'));
```

Comment:
Doesn't this toggle have a proper label? Use labels or other actually visible texts whenever possible - only use test-ids if nothing else is an option (and even then, rather add a label than using test ids).

Reply:
Set the `aria-label` and used `getByLabelText` for aspect ratio lock and padding ratio lock.